lib/net.js

```diff
@@ -59,47 +58,57 @@ exports.createServer = function(options, connectionListener) {
 // connect(port, [host], [cb])
 // connect(path, [cb]);
 //
-exports.connect = exports.createConnection = function() {
-  var args = new Array(arguments.length);
-  for (var i = 0; i < arguments.length; i++)
-    args[i] = arguments[i];
-  args = normalizeArgs(args);
-  debug('createConnection', args);
-  var s = new Socket(args[0]);
+exports.connect = exports.createConnection = function(...args) {
```

Comment:
I was thinking it more had to do with the number of arguments passed or something along those lines...